### PR TITLE
Support Terraform 1.0.0

### LINKED ISSUE #89: Terraform 1.0.0 support
Fails on 1.0.0

```
terratag -tags='{ "source_repo": "corp/repo/path" }' -verbose-true
2021/06/14 13:11:47 Unable to parse 'terraform version'

❯ terraform version
Terraform v1.0.0
on darwin_amd64
+ provider registry.terraform.io/hashicorp/aws v3.45.0
```

Thanks!

### PR DESCRIPTION
**Pull request by roni-frantchi**
Resolves #89

### CODE REVIEW

**Comment by roni-frantchi** [comment]
[Terratag fails on TF v1.0.0](https://trello.com/c/FfuAqVar/3481-terratag-fails-on-tf-v100)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -50,5 +50,19 @@ jobs:
...
+          TF_VERSION: ${{ matrix.terraform_version }}
+        uses: actions/github-script@v4
+        with:
+          script: |
+            console.log(`Setting test suite version based on ${process.env.TF_VERSION}`)
+            const suite = process.env.TF_VERSION.replace('.', 'o')
+            console.log(`Test suite set to ${suite}`)
+            return suite
+          result-encoding: string
+
```
Comment (avnerenv0) [inline .github/workflows/ci.yml:65]: wow this is nice !

File: terratag_test.go [hunk 2]
```diff
@@ -44,7 +44,11 @@ func TestTerraform14(t *testing.T) {
 }
 
 func TestTerraform15(t *testing.T) {
-	testTerraform(t, "15")
+	testTerraform(t, "15_1.0")
+}
+
+func TestTerraform1o0(t *testing.T) {
+	testTerraform(t, "15_1.0")
```
Comment (avnerenv0) [inline terratag_test.go:51]: how come the tests for 1 and 15 are the same?

File: convert/convert.go [hunk 3]
```diff
@@ -49,8 +49,8 @@ func trimTokens(tokens hclwrite.Tokens) hclwrite.Tokens {
 	return tokens[startIndex : len(tokens)-endIndex]
 }
 
-func buildMapExpression(tokens hclwrite.Tokens, tfVersion int) string {
-	if tfVersion >= 15 {
+func buildMapExpression(tokens hclwrite.Tokens, tfVersion Version) string {
+	if tfVersion.Major == 0 && tfVersion.Minor >= 15 || tfVersion.Major == 1 {
```
Comment (avnerenv0) [inline convert/convert.go:53]: https://github.com/env0/terratag/issues/79 :(

**Review by avnerenv0 — APPROVED** [review]
👍

File: terratag_test.go (hunk 2 above)
Comment (roni-frantchi) [reply terratag_test.go:51]: We do the same between 13 and 14.
Basically nothing has changed in terms of in/out syntax and no new cases, so we reuse the fixture files

**Pull request merged**